src/main/assembly/assemble-connector.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update the license header as follows? Please update the rest of the files as well
```suggestion
   Copyright (c) 2022, WSO2 LLC (https://www.wso2.com) All Rights Reserved.
```